lib/internal/Magento/Framework/View/Result/Layout.php

```diff
@@ -7,6 +7,7 @@
 namespace Magento\Framework\View\Result;
 
 use Magento\Framework;
+use Magento\Framework\App\Response\HttpInterface as HttpResponseInterface;
```

Comment:
After this change we started getting the following static tests failure:
`The class Layout has a coupling between objects value of 13. Consider to reduce the number of dependencies under 13.`